? src/all.src
? src/appendsrc.sh
? src/icecube.daq.stf.autogen.cf
? src/icecube.daq.stf.autogen.deps
? src/nlines.sh
Index: private/stf-apps/atwd_pedestal_noise.c
===================================================================
RCS file: /home/icecube/cvsroot/stf/private/stf-apps/atwd_pedestal_noise.c,v
retrieving revision 1.3
diff -u -r1.3 atwd_pedestal_noise.c
--- private/stf-apps/atwd_pedestal_noise.c	24 Jun 2004 00:55:21 -0000	1.3
+++ private/stf-apps/atwd_pedestal_noise.c	3 Jun 2005 09:31:19 -0000
@@ -16,6 +16,7 @@
 }
 
 BOOLEAN atwd_pedestal_noiseEntry(STF_DESCRIPTOR *d,
+                    unsigned integrated,
                     unsigned atwd_sampling_speed_dac,
                     unsigned atwd_ramp_top_dac,
                     unsigned atwd_ramp_bias_dac,
@@ -121,9 +122,11 @@
       *noise_rms = (unsigned) (rms*1000);
    }
 
-   return 
-      avg > 95 && avg < 105 &&
-      rms < 2.0 &&
-      *noise_negative_max < 10 &&
-      *noise_positive_max < 10;
+   {  const int maxlimit = (integrated) ? 20 : 10;
+      
+      return 
+         avg > 95 && avg < 105 &&
+         rms < 2.0 &&
+         *noise_negative_max < maxlimit && *noise_positive_max < maxlimit;
+   }
 }
